src/tests/ftest/pool/RebuildTests.py

```diff
@@ -236,118 +201,69 @@ def test_multipool_rebuild(self):
             pool1.connect(1 << 1)
             pool2.connect(1 << 1)
 
             # create containers
-            container1 = DaosContainer(self.Context)
+            container1 = DaosContainer(self.context)
             container1.create(pool1.handle)
-            container2 = DaosContainer(self.Context)
+            container2 = DaosContainer(self.context)
             container2.create(pool2.handle)
 
             # now open them
             container1.open()
             container2.open()
 
-            # how many objects and records are we creating
-            objcount = self.params.get("objcount",
-                                       '/run/testparams/numobjects/*')
-            reccount = self.params.get("reccount",
-                                       '/run/testparams/numrecords/*')
-            if objcount == 0:
-                reccount = 0
-
-            # which rank to write to and kill
-            rank = self.params.get("rank", '/run/testparams/ranks/*')
-
-            # how much data to write with each key
-            size = self.params.get("size", '/run/testparams/datasize/')
-
             # Putting the same data in both pools, at least for now to simplify
             # checking its correct
             saved_data = []
-            for i in range(0, objcount):
+            for _objc in range(self.objcount):
                 obj = None
-                for j in range(0, reccount):
+                for _recc in range(self.reccount):
 
                     # make some stuff up and write
-                    dkey = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
+                    dkey = ''.join(random.choice(string.ascii_uppercase + string.digits)
                                    for _ in range(5))
-                    akey = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
+                    akey = ''.join(random.choice(string.ascii_uppercase + string.digits)
                                    for _ in range(5))
-                    data = ''.join(random.choice(string.ascii_uppercase +
-                                                 string.digits)
-                                   for _ in range(size))
-
-                    obj, tx = container1.write_an_obj(data, len(data),
-                                                         dkey, akey, obj, rank)
-                    obj, tx = container2.write_an_obj(data, len(data),
-                                                         dkey, akey, obj, rank)
-
+                    data = ''.join(random.choice(string.ascii_uppercase + string.digits)
+                                   for _ in range(self.size))
+
+                    #Used DAOS_OC_R1S_SPEC_RANK,    /* 1 replica with specified rank */
+                    obj, tx = container1.write_an_obj(data, len(data), dkey, akey, obj, self.rank,
```

Comment:
(pylint-line-too-long) Line too long (98/80)

(pylint-invalid-name) Invalid variable name "tx"

Reply:
no belong to current patch.